src/cartridges/int_adyen_overlay/cartridge/scripts/handleCustomObject.ds

```diff
@@ -136,9 +133,11 @@ function handle( customObj : CustomObject  ) : Object
 				break;
 			//CustomAdyen 
 			case "CAPTURE_FAILED" :
-				order.setPaymentStatus(Order.PAYMENT_STATUS_NOTPAID);
-				order.setExportStatus(Order.EXPORT_STATUS_NOTEXPORTED);
-				OrderMgr.cancelOrder(order);
+				if(customObj.custom.success !== "true") {
```

Comment:
shouldn't be?

```suggestion
				if(customObj.custom.success == "true") {
```

Reply:
but this is CAPTURE_FAILED so if the CAPTURE_FAILED success is false (which I don't even know if can happen) then it is captured, isn't it?

Reply:
It indeed needs to be: if(customObj.custom.success == "true") 

CAPTURE_FAILED success=true indicates that the capture failed